enterprise/causal-clustering/src/main/java/org/neo4j/causalclustering/core/CausalClusteringSettings.java

```diff
@@ -101,10 +102,21 @@
     public static final Setting<Integer> raft_in_queue_max_batch =
             setting( "causal_clustering.raft_in_queue_max_batch", INTEGER, "64" );
 
-    @Description( "Expected number of Core machines in the cluster" )
+    @Description( "Expected number of Core machines in the cluster before startup" )
+    @Deprecated
+    @ReplacedBy( "minimum_core_cluster_size_at_startup and minimum_core_cluster_size_at_runtime" )
     public static final Setting<Integer> expected_core_cluster_size =
             setting( "causal_clustering.expected_core_cluster_size", INTEGER, "3" );
 
+    @Description( "Minimum number of Core machines in the cluster at startup" )
```

Comment:
Perhaps formation instead of startup? In actuality it is for formation of the underlying hazelcast cluster. The majority of the members must be available to select a hazelcast master. I think minimum doesn't perhaps describe this majority quorum behaviour sufficiently.

Reply:
Done.